.github/workflows/sdks.yml

```diff
@@ -209,16 +209,10 @@ jobs:
           SDK=`pwd`/$SDK_NAME
 
           git apply swift-android.patch swift-android-foundation.patch swift-android-ci.patch swift-crypto.patch swift-system.patch
-          if [ ${{ matrix.version }} = 'trunk' ]; then
-            git apply swift-android-ci-trunk.patch swift-android-foundation-trunk.patch
+          if [ ${{ matrix.version }} = 'release' ]; then
+              git apply swift-android-ci-release.patch swift-android-foundation-release.patch
           else
-            git apply swift-android-ci-except-trunk.patch swift-android-foundation-except-trunk.patch
-            git apply -C1 swift-android-foundation-armv7-except-trunk.patch
-            if [ ${{ matrix.version }} = 'devel' ]; then
-              perl -pi -e 's%Musl%Bionic%g' swift-corelibs-foundation/Sources/Testing/Testing.swift
-            else
-              git apply swift-android-foundation-release.patch
-            fi
+            git apply swift-android-ci-except-release.patch swift-android-foundation-except-release.patch
```

Comment:
Now that Swift 6.1 has branched, this is much simpler, @marcprux, let me know what you think.

Of course, as trunk starts to diverge, these patches will again too, so I'll push to get these in.

Reply:
Nice! Much simpler.